packages/amplify-cli/tsconfig.json

```diff
@@ -39,4 +37,4 @@
       "path": "../amplify-cli-shared-interfaces"
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
as a team we keep adding and removing extra new line, we should probably get linter rule for this.